docs/user/pings/baseline.md

```diff
@@ -3,21 +3,21 @@
 ## Description
 
 This ping is intended to provide metrics that are managed by the library itself, and not explicitly set by the application or included in the application's `metrics.yaml` file.
-If the application crashes no `baseline` ping is sent, no additional ping is generated with the data from before the crash.
 
 > **Note:** As the `baseline` ping was specifically designed for mobile operating systems, it is not sent when using the Glean Python bindings.
 
 ## Scheduling
 
-The `baseline` ping is automatically sent when the application is moved to the [background](index.md#defining-background-state).
+The `baseline` ping is automatically submitted with a `reason: background` when the application is moved to the [background](index.md#defining-background-state).
+The `baseline` ping will be submitted at startup with a `reason: dirty_startup`, if the previous session was not cleanly closed. This only happens from the second start.
```

Comment:
Just as a clarification: `reasons` will end up as normal text fields in the table? Therefore whether to use underscores or dashes is just a stylistic question?

Reply:
Yes, exactly.